vendor/github.com/containers/kubensmnt/kubensmnt_linux.go

```diff
@@ -0,0 +1,50 @@
+//go:build linux && cgo
+// +build linux,cgo
+
```

Comment:
Not related to this PR but these build tags are redundant.

Reply:
My editor adds both of them automagically for backwards-compatibility reasons.